src/widget/weffectparameternamebase.cpp

```diff
@@ -91,12 +97,19 @@ void WEffectParameterNameBase::parameterUpdated() {
                           metrics.size(0, m_text).width()) +
             2 * frameWidth();
     setText(m_text);
-    m_parameterUpdated = true;
+    // valueChanged() is also emitted after a new parameter has been loaded if
+    // that is linked to the Meta knob.
+    // We set a flag in order to not show the value in that case.
+    // Though only suppress it for those knobs! For unlinked knobs it would
+    // prevent showing the value on first click.
+    EffectKnobParameterSlot* paramKnob =
```

Comment:
Btw this independent from the Meta knob keep/reset setting, ie. now works as expected in all situations.